<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoRejestracji</class>
 <widget class="QDialog" name="OknoRejestracji">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>270</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Rejestracja</string>
  </property>
  <property name="whatsThis">
   <string>OknoRejestracji</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="12" column="3">
    <widget class="QPushButton" name="buttonRegister">
     <property name="text">
      <string>Rejestruj</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="labelHaslo">
     <property name="text">
      <string>Haslo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leHaslo</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="3">
    <widget class="QLineEdit" name="leNick">
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <widget class="QLineEdit" name="leHaslo">
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelNick">
     <property name="text">
      <string>Pseudonim</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leNick</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QLabel" name="labelLoginError">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>10</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leLogin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QTextEdit" name="teTytul">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>60</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Podaj dane logowania oraz pseudonim pod którym będziesz widoczny&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="3">
    <widget class="QLabel" name="labelHasloError">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>10</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leHaslo</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="3">
    <widget class="QLabel" name="labelNickError">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>10</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leNick</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelLogin">
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leLogin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="leLogin">
     <property name="maxLength">
      <number>32</number>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>leHaslo</zorder>
  <zorder>leNick</zorder>
  <zorder>buttonCancel</zorder>
  <zorder>leLogin</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>buttonRegister</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>labelHaslo</zorder>
  <zorder>labelNick</zorder>
  <zorder>teTytul</zorder>
  <zorder>labelLoginError</zorder>
  <zorder>labelHasloError</zorder>
  <zorder>labelNickError</zorder>
  <zorder>labelLogin</zorder>
 </widget>
 <tabstops>
  <tabstop>leLogin</tabstop>
  <tabstop>leHaslo</tabstop>
  <tabstop>leNick</tabstop>
  <tabstop>buttonRegister</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>teTytul</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>OknoRejestracji</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLogin</sender>
   <signal>textChanged(QString)</signal>
   <receiver>labelLoginError</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leHaslo</sender>
   <signal>textChanged(QString)</signal>
   <receiver>labelHasloError</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNick</sender>
   <signal>textChanged(QString)</signal>
   <receiver>labelNickError</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
